odoo/tools/mimetypes.py

```diff
@@ -26,6 +26,240 @@
     'pt/': 'application/vnd.openxmlformats-officedocument.presentationml.presentation',
     'xl/': 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
 }
+
+_Signature = collections.namedtuple('_Signature', ['magic_number', 'offset'])
```

Comment:
I'm usually a fan of `collections.namedtuple` but in this case since most magic numbers are at the beginning of the file, I think it is worth we use a default value for the offset.
```suggestion
class _Signature(typing.NamedTuple):
    magic_number: bytes
    offset: int = 0
```

Reply:
And so drop `offset=0` from all places where the default is fine, and use a named argument for the few other places